test/unit/test_sdb.h

```diff
@@ -0,0 +1,21 @@
+
+#ifndef TEST_SDB_H
+#define TEST_SDB_H
+
+static void diff_cb(const SdbDiff *diff, void *user) {
```

Comment:
what about just putting these in min_unit.h?

Reply:
I'm not a huge fan of this small extra-header either, but putting it in `minunit.h` would mean that every test must include `sdb.h`.

Reply:
Good point yeah. Though any unit test probably needs sdb.h anyway, because sdb.h is required by RUtil and anything in r2world requires RUtil.